pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -521,15 +519,15 @@ def apply_lightning(self, operations):
             """
             # Skip over identity operations instead of performing
             # matrix multiplication with the identity.
-            skipped_ops = ["Identity"]
-            invert_param = False
             for ops in operations:
-                if str(ops.name) in skipped_ops:
```

Comment:
what was this casting to `str` suppose to resolve?

Reply:
it may be required for tensor product ops :thinking:

Reply:
This is not required at this point of the workflow as far as I know. It is not in `lightning.qubit` either, so I think it can be removed to bring the implementations inline with one another.